networkx/generators/tests/test_line.py

```diff
@@ -275,3 +275,9 @@ def test_line_different_node_types(self):
         H = nx.line_graph(G)
         J = nx.inverse_line_graph(H)
         assert nx.is_isomorphic(G, J)
+
+    def test_line_inverse_edgeless_graph(self):
+        G = nx.Graph()
+        G_nodes = [1, 2, 3, "wilber", "seth"]
```

Comment:
Could you please remove my name from here ? :)

It's better to use generic strings instead of names of contributors.

Reply:
I have updated, thank you!

Reply:
> Could you add the link to the exact issue this pull request is fixing?

https://github.com/networkx/networkx/issues/6028